src/components/clients/ClientList.js

```diff
@@ -14,7 +14,9 @@ class ClientList extends React.Component {
         <table className="table">
           <thead>
             <tr>
-              {['Name', 'API key'].map(title => {
+              {['Name', 'API key',
+                'Auth redirect URL',
+                'Auth failure redirect URL'].map(title => {
                 return <th key={title}>{title}</th>
               })}
```

Comment:
nit: could be somewhat more idiomatic to use the shorter form of the arrow function here: 
```javascript
[ ...
  ...
  ...].map(
    title => <th key={title}>{title}</th>
  )
```

up yo tou though :)